mithril-aggregator/src/tools/genesis.rs

```diff
@@ -145,7 +145,7 @@ impl GenesisTools {
         to_sign_payload_file.read_to_end(&mut to_sign_payload_buffer)?;
 
         let genesis_signature = genesis_signer.sign(&to_sign_payload_buffer);
-        let signed_payload = genesis_signature.to_bytes();
+        let signed_payload = genesis_signature.key().to_bytes();
```

Comment:
Idea: we could maybe implement `Deref` trait for the type so that we don't need to update this code?